src/main/java/com/limechain/Main.java

```diff
@@ -41,10 +52,12 @@ public static void main(String[] args) {
 
         // Start the client
         // NOTE: This starts the beans the client would need - mutates the global context
+        prometheusServer.emitStartTime();
```

Comment:
This is plugged in between the comment and the line it relates to. I'd move it above or below.